global_install_scripts/install.bash

```diff
@@ -408,7 +408,10 @@ END_OF_SCRIPT
 	setup_git_template_talisman_hook
 	echo
 	echo "Setting up talisman hook recursively in git repos"
-	read -p "Please enter root directory to search for git repos (Default: ${HOME}): " SEARCH_ROOT
+	
+	if [[ "$SEARCH_ROOT" == "" ]]; then
+		read -p "Please enter root directory to search for git repos (Default: ${HOME}): " SEARCH_ROOT
+	fi
 	SEARCH_ROOT=${SEARCH_ROOT:-$HOME}
```

Comment:
The installation section of the readme should also be updated so users know SEARCH_ROOT can be configured.

Reply:
https://github.com/thoughtworks/talisman/pull/349/commits/d8f87ab610388aa6694f0a9c74de7f2ec1de2479